server_metrics.go

```diff
@@ -61,6 +62,24 @@ func WithHistogramBuckets(buckets []float64) HistogramOption {
 	return func(o *prom.HistogramOpts) { o.Buckets = buckets }
 }
 
+func WithHistogramConstLabels(labels prom.Labels) HistogramOption {
```

Comment:
Why this function is not in metric_options.go and not used?

Reply:
I found the other `WithXXX() HistogramOption` functions in `server_metrics.go`, so I decided to put my one here to. You are right, they are all not specific to server instrumentation, so I would move all funcs returning a 
 `HistogramOption` to metric_options.go if you agree.

Why they're not used? I hope I get your question right: all of the `WithXXX()` functions are not used. They are functions a package user can use to configure the metrics.

Reply:
I see. I prefer to move all WithXXX() option to metric_options.go.
Writing these functions in this file feels confusing.

I asked you why not used this function because I only saw New{Sever,Client}Metrics function. I got now, it is wrong comment. Sorry.